Sources/SotoCore/AWSClient.swift

```diff
@@ -466,26 +476,30 @@ extension AWSClient {
     ///     - httpHeaders: Headers that are to be used with this URL. Be sure to include these headers when you used the returned URL
     ///     - expires: How long before the signed URL expires
     ///     - serviceConfig: additional AWS service configuration used to sign the url
-    ///     - logger: Logger to output to
+    ///     - context: Baggage context, holding a logger and baggage for tracing
     /// - returns:
     ///     A signed URL
     public func signURL(
         url: URL,
         httpMethod: HTTPMethod,
         headers: HTTPHeaders = HTTPHeaders(),
-        expires: Int = 86400,
-        serviceConfig: AWSServiceConfig,
-        logger: Logger = AWSClient.loggingDisabled
+        config: AWSServiceConfig,
+        context: Context = AWSClient.emptyContext,
+        expires: Int = 86400
```

Comment:
to be honest I find it a bit strange that expires has a default value, is it for consistency with other (official) AWS frameworks?

most API has "input" before the context, so the signature is a bit inconsistent here

Reply:
Yeah you are probably correct here. I think I added the default in the signer and it just bubbled through all the APIs. I'm gonna make this a `TimeAmount` as well. Makes for easier reading, but will do in separate PR though

Reply:
See #373